rules.bzl

```diff
@@ -102,9 +124,11 @@ UBUNTU16_04_IMAGE = "gcr.io/flame-public/executor-docker-default:enterprise-v1.6
 
 UBUNTU20_04_IMAGE = "gcr.io/flame-public/rbe-ubuntu20-04:latest"
 
+UBUNTU22_04_ARM64_IMAGE = "gcr.io/flame-public/rbe-ubuntu22-04:latest-arm64"
```

Comment:
what commit was this built from? I can help do a one-off build for an amd64 image so that we can start with a cross-platform image right off the bat, instead of having to migrate later.

Reply:
I have a cross-platform image, but if we use it oci resolve says it can't pick an image :/

Reply:
The error in question, which I am trying to solve: https://buildbuddy.buildbuddy.io/invocation/4be9aa05-5736-4ddd-a4f0-17a9c83544c6

Reply:
It didn't work with the busybox image either, if you recall. I think there's a problem with `Resolve` in `enterprise/server/util/oci/oci.go`, and it probably has to do with `Variant`s :(

Reply:
I re-sent the patch here: https://github.com/buildbuddy-io/buildbuddy/pull/7744

Reply:
I switched us to using a multi-platform image by default